Docs/feb maintenance 05 (#466)
rsksmart/devportal: solutions/mathwallet.md

---
title: 'Math Wallet - Solutions on RSK'
description: 'CryptoWallet for 1 million users, 50+ blockchains, 2000+ dApps, 200+ DeFi.'
tags: Math Wallet, rsk-solutions, rsk, wallet, bitcoin, defi, dApps, blockchain, cryptowallet
render_features: 'tables-with-borders'
layout: 'rsk'
---

![Math Wallet- BannerImage](/assets/img/solutions/MathWallet/MathWallet_Logo_Black.png)

## Description

> Math Wallet is a CryptoWallet used by 1 million users, 50+ blockchains, 2000+ dApps, 200+ DeFi. It's currently supported tokens are: ETH, Polkadot, Near, Solana, Binance Smart Chain, Filecoin, EOS, TRON, Binance, cosmos, Irisnet, Ontology ,Kulupu, chainX, Kusama, Edgeware, EOS Force, Enigma, Secret network, Nervos, vechain, FTchain, Harmony, Vapor, Zilliqa, YAS, RSK, Enumivo, BOS, Telos, NEO, Nebulas, ESN.

Reviewer: ```suggestion
Math Wallet is a CryptoWallet used by 1 million users, 50+ blockchains, 2000+ dApps, 200+ DeFi. It's currently supported networks include: RSK, ETH, Polkadot, Near, Solana, Binance Smart Chain, Filecoin, EOS, TRON, Binance, cosmos, Irisnet, Ontology, Kulupu, chainX, Kusama, Edgeware, EOS Force, Enigma, Secret network, Nervos, vechain, FTchain, Harmony, Vapor, Zilliqa, YAS, Enumivo, BOS, Telos, NEO, Nebulas, ESN.
```

## How Math Wallet integrates with RSK

![MathWallet-RSK-Integrations](/assets/img/solutions/MathWallet/Diagram-MathWallet.png)

### Supported Tokens

Math Wallets supports;

- BTC
- ETH
- BNB
- DOT
- NEAR
- FILECOIN
- EOS
- TRON
- IRIS
- COSMOS
- ONTPCX
- KLP
- others

> 
> 
> ### Supported Token Standards

Reviewer: ```suggestion

### Supported Token Standards
```

| [DAI](https://etherscan.io/token/0x6b175474e89094c44da98b954eedeac495271d0f) | [Dai](https://makerdao.com/ ) | ERC20 | Ethereum
| [UNI](https://etherscan.io/token/0x1f9840a85d5af5bf1d1762f925bdaddc4201f984) | [Uniswap](https://uniswap.org/) | ERC20 | Ethereum
| [USDT](https://etherscan.io/token/0xdac17f958d2ee523a2206206994597c13d831ec7) | [Tether USD](https://tether.to/)  | ERC20 | Ethereum
| [SAI](https://etherscan.io/token/0x89d24a6b4ccb1b6faa2625fe562bdd9a23260359) | [Sai Stablecoin](https://makerdao.com/)   | ERC20 | Ethereum
| [TUSD](https://etherscan.io/token/0x0000000000085d4780B73119b644AE5ecd22b376) | [TrueUSD](https://www.trusttoken.com/trueusd/) | ERC20 | Ethereum
| [GUSD](https://etherscan.io/token/0x056fd409e1d7a124bd7017459dfea2f387b6d5cd) | [Gemini dollar](https://gemini.com/dollar/) | ERC20 | Ethereum
| [PAX](https://etherscan.io/token/0x8e870d67f660d95d5be530380d0ec0bd388289e1) | [Paxos Standard](https://www.paxos.com/standard) | ERC20 | Ethereum
| [USDC](https://etherscan.io/token/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48) | [USD Coin](https://www.centre.io/) | ERC20 | Ethereum

> 
> ### Non-Fungible tokens
> 
> | Symbol | Name | Token Standard | Network
> | --- | --- | --- | --- |
> | [AXIE](https://etherscan.io/token/0xf5b0a3efb8e8e4c201e2a935f110eaaf3ffecb8d) | [Axie](https://axieinfinity.com/) | ERC721 | Ethereum

Reviewer: ```suggestion
```

> 
> ### Non-token financial assets
> 
> | Symbol | Name | Network
> | --- | --- | --- |
> | [BTCX](https://explorer.rsk.co/address/0xf773b590af754d597770937fa8ea7abdf2668370) | [BTCX](https://moneyonchain.com/btcx-leveraged-bitcoin/) | RSK

Reviewer: ```suggestion
```
